modelexpress_common/src/config.rs

```diff
@@ -61,6 +60,16 @@ impl fmt::Display for DurationConfig {
     }
 }
 
+// Serialize as just the number of seconds (not as a struct)
+impl Serialize for DurationConfig {
```

Comment:
was it the missing `Serialize` trait that was causing the problem? (im not really familiar with serde in rust)